tools/buildPackage.js

```diff
@@ -19,11 +19,24 @@ var env = {
   NODE_ENV: 'production'
 }
 
+console.log('Cleaning build target')
+
+if (isWindows) {
+  var cmds = [
+    'if exist ' + buildDir + ' rmdir /s /q ' + buildDir,
+  ]
+  execute(cmds, env, console.log.bind(null, 'done'))
+} else {
+  var cmds = [
+    'rm -rf ' + buildDir,
+    'rm -f dist/*.dmg dist/*.nupkg dist/*.exe dist/*.msi dist/RELEASES dist/*.zip',
+  ]
+  execute(cmds, env, console.log.bind(null, 'done'))
+}
```

Comment:
I played around with push/append. Looks like `.append()` (for adding multiple elements to an array) doesn't exist in this version of js; i'm already using `.concat()` in my latest commit, so as long as the overall approach doesn't grate, it should work!